src/hotkey/hotkey_handler.cpp

```diff
@@ -272,6 +272,7 @@ bool play_controller::hotkey_handler::can_execute_command(const hotkey::hotkey_c
 			return true;
 		}
 	}
+	size_t humans_notme_cnt = 0;
```

Comment:
I am checking out a new tool which says this variable (but not the one below on line 328) is not used.

Verify and remove if this is correct.

Reply:
I had moved the variable into a smaller scope in the case.

Since this PR was turned back into the mess it was before my force push the variable declaration is now duplicated.

Reply:
i don't understand

Reply:
Why don't you (@kallaballa) add him (@soliton-) as a contributor on your fork and let him clean up the PR and force-push it?